.github/workflows/linux.yml

```diff
@@ -7,7 +7,19 @@ on:
     types: published
 
 jobs:
+  skip-duplicates:
```

Comment:
You shouldn't need to have this be its own job.

I'd prefer just adding the step to the existing job(s) before the checkout step.

Is that possible?